lib/widgets/compose_box.dart

```diff
@@ -274,13 +274,15 @@ class ComposeContentController extends ComposeController<ContentValidationError>
 
 class _ContentInput extends StatefulWidget {
   const _ContentInput({
+    required this.enabled,
```

Comment:
Yeah, I assumed the size/position change is a typo, as the baseline of the text does move in Figma: ![image](https://github.com/user-attachments/assets/8c6e3868-650e-4582-b2e9-aef14a7f2ae9)
We can still move the border up 1px and render the loading indicator below it, without also moving the text. Maybe that's something we can ask in chat for clarification.

> The second bullet point doesn't make sense to me because (if I'm reading correctly) all of the buttons are faded in the new disabled state, and that matches the Figma. There's an implementation detail about how that gets done (with an Opacity vs. something else), but I don't see an inconsistency with the Figma to mention here.

Makes sense. I was referring to how we apply opacity to the entire compose button bar and etc. Let's leave this one out.

Reply:
> the baseline of the text does move in Figma

Oh, that is subtle! Thanks for pointing that out, and also that the border moves up 1px.

I agree it looks accidental. It looks like something is adding 1px of height to the compose box as empty space at the bottom.

Looks like the thing doing that is a "spacing bar" element. It looks like all these frames have such an element, and most of them use it to pad the bottom inset. But in frames where the keyboard shows, it's meant to be a no-op because the keyboard is already padding the bottom inset; we don't want to double-pad it. In this particular frame (with the loading indicator) the element fails to be a no-op; its top edge sticks out from under the keyboard by 1px, unlike in the adjacent frame where the top of the element is aligned with the top of the keyboard.

| [Frame on the left](https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=4031-8290&node-type=instance&m=dev) | [Frame on the right](https://www.figma.com/design/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=4031-8567&node-type=instance&m=dev) |
| --- | --- |
| <img width="327" alt="image" src="https://github.com/user-attachments/assets/7a0c543d-4b50-4d41-9b40-8717e18c82c4"> | <img width="326" alt="image" src="https://github.com/user-attachments/assets/26d7385b-8d6d-4d6a-86f8-abcbe187bed2"> |